lib/src/json_schema/json_schema.dart

```diff
@@ -548,6 +548,19 @@ class JsonSchema {
     return baseSchema;
   }
 
+  /// Look for the given anchor at the schema. Returns null if nothing is found.
+  JsonSchema _resolveDynamicAnchor(JsonSchema schema, String anchor) {
+    // IDs in draft2019 and up do not have fragments.
```

Comment:
should this bail out early on earlier drafts and return null?

Reply:
It could. It should also never be called on drafts before 2020-12 because that is the first draft that supports the `dynamicAnchor` keyword.